### DISCUSSION TITLE
chatwoot cluster architecture

### DISCUSSION THREAD

**Discussion by zell92** [chatwoot/chatwoot#2588] [Q&A] [answered]
Hello everybody. In the documentation I could not find any resolution for my doubt: is it possible to install chatwoot (self hosted) in a cluster architecture with two servers?
and if it is possible, is there a guide I can follow?
Thanks so much

**Comment by sojan-official** [comment, marked as answer]
Please refer https://www.chatwoot.com/docs/self-hosted/deployment/architecture 

You can run multiple chatwoot web and worker servers, connect them to a single Postgres/Redis cluster. Then route the traffic between the available web servers using a load balancer. 

ref: https://github.com/chatwoot/chatwoot/discussions/2593
